packages/portfolio-contract/test/snapshots/portfolio.contract.test.ts.md

```diff
@@ -380,6 +380,122 @@ Generated by [AVA](https://avajs.dev).
+
+    {
+      'Durable Publish Kit_kindHandle': 'Alleged: kind',
+      Recorder_kindHandle: 'Alleged: kind',
+      asyncFlow: {
+        AdminAsyncFlow_kindHandle: 'Alleged: kind',
+        AdminAsyncFlow_singleton: 'Alleged: AdminAsyncFlow',
+        Bijection_kindHandle: 'Alleged: kind',
+        FunctionUnwrapper_kindHandle: 'Alleged: kind',
+        FunctionUnwrapper_singleton: 'Alleged: FunctionUnwrapper',
+        LogStoreMetadata: {
+          generation: 0,
+        },
+        LogStore_kindHandle: 'Alleged: kind',
+        StateUnwrapper_kindHandle: 'Alleged: kind',
+        asyncFuncEagerWakers: [],
+        asyncFuncFailures: {},
+        flowForOutcomeVow: {},
+        unwrapMap: 'Alleged: weakMapStore',
+      },
+      contract: {
+        PortfolioPub_kindHandle: 'Alleged: kind',
+        PortfolioPub_singleton: 'Alleged: PortfolioPub',
+        Portfolio_kindHandle: 'Alleged: kind',
+        Position_kindHandle: 'Alleged: kind',
+        orchestration: {
+          openPortfolio: {
+            asyncFlow_kindHandle: 'Alleged: kind',
+            endowments: {
+              0: {
+                inertSubscriber: {
+                  getUpdateSince_kindHandle: 'Alleged: kind',
+                  getUpdateSince_singleton: 'Alleged: getUpdateSince',
+                  subscribeAfter_kindHandle: 'Alleged: kind',
+                  subscribeAfter_singleton: 'Alleged: subscribeAfter',
+                },
+                makePortfolioKit_kindHandle: 'Alleged: kind',
+                makePortfolioKit_singleton: 'Alleged: makePortfolioKit',
+                zoeTools: {
+                  localTransfer_kindHandle: 'Alleged: kind',
+                  localTransfer_singleton: 'Alleged: localTransfer',
+                  withdrawToSeat_kindHandle: 'Alleged: kind',
+                  withdrawToSeat_singleton: 'Alleged: withdrawToSeat',
+                },
+              },
+            },
+          },
+          rebalance: {
+            asyncFlow_kindHandle: 'Alleged: kind',
+            endowments: {
+              0: {
+                zoeTools: {
+                  localTransfer_kindHandle: 'Alleged: kind',
+                  localTransfer_singleton: 'Alleged: localTransfer',
+                  withdrawToSeat_kindHandle: 'Alleged: kind',
+                  withdrawToSeat_singleton: 'Alleged: withdrawToSeat',
+                },
+              },
+            },
+          },
+          rebalanceFromTransfer: {
+            asyncFlow_kindHandle: 'Alleged: kind',
+            endowments: {
+              0: {
+                zoeTools: {
+                  localTransfer_kindHandle: 'Alleged: kind',
+                  localTransfer_singleton: 'Alleged: localTransfer',
+                  withdrawToSeat_kindHandle: 'Alleged: kind',
+                  withdrawToSeat_singleton: 'Alleged: withdrawToSeat',
+                },
+              },
+            },
+          },
+        },
+        portfolios: {
+          0: {
```

Comment:
why doesn't this capture state shape?

Reply:
The short answer is that it's not in baggage but rather a different part of the vat store, cf. #11694. I think what you really want is #10950.

Reply:
looks like what we see for portfolios is consistent with what we see for [snapshot of Fast USDC OCW operators](https://github.com/Agoric/agoric-sdk/blob/master/packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md):

```
    operators: {
      'operator-0': {
        admin: Object @Alleged: Operator Kit admin {},
        invitationMakers: Object @Alleged: Operator Kit invitationMakers {},
        operator: Object @Alleged: Operator Kit operator {},
      },
```